code/modules/mob/living/simple_animal/hostile/human/surgeon.dm

```diff
@@ -0,0 +1,69 @@
+/mob/living/simple_animal/hostile/humanoid/surgeon/
+	name = "\improper Surgeon"
+	desc = "Don't interfere with his work!"
+	icon = 'icons/mob/hostile_humanoid.dmi'
+	icon_state = "surgeon"
+	icon_living = "surgeon"
+	maxHealth = 100
+	health = 100
+
+	speed = 1
+	harm_intent_damage = 10
+	melee_damage_lower = 8
+	melee_damage_upper = 13
+	attacktext = "saws"
+	attack_sound = 'sound/weapons/circsawhit.ogg'
+
+	ranged = 0
+	gender = MALE
+
+	//corpse = /obj/effect/landmark/corpse/surgeon
+	items_to_drop = list(/obj/item/weapon/circular_saw/plasmasaw)
+
+/mob/living/simple_animal/hostile/humanoid/surgeon/normal
+	name = "Doctor Sawyer"
+	desc = "A deranged surgeon lost deep in space."
+	icon_dead = null //this isn't even his final form
+	maxHealth = 200
+	health = 200
+
+	speed = 1
+	harm_intent_damage = 20
+	melee_damage_lower = 17
+	melee_damage_upper = 23
+
+	corpse = null //he isn't dead yet
+
+	faction = "skeleton" //don't want him to get killed by skeletons before his transformation
+
+/mob/living/simple_animal/hostile/humanoid/surgeon/normal/Die()
+	src.say("[pick("YOU CAN'T KILL ME THAT EASILY!", "I WONT LET YOU STOP ME!", "GET OUT OF MY FACILITY!", "I MUST CONTINUE MY RESEARCH!", "I'M GONNA WRECK IT!", "I'VE GOT A BONE TO PICK WITH YOU!")]")
+	var/obj/effect/effect/smoke/S = new /obj/effect/effect/smoke(get_turf(src))
+	to_chat("<span class=danger><B>Before he can die, the mad surgeon takes a drink of a foul-smelling concoction and begins to mutate! </span></B>")
+	S.time_to_live = 20
+	src.
+
+	..()
+	return qdel(src)
+
+/mob/living/simple_animal/hostile/humanoid/surgeon/skeleton
+	name = "Skeletal Strongman"
+	desc = "He wont be pushed around any longer"
+	//icon_state = "skeletonboss"
+	//icon_living = "skeletonboss"
+	maxHealth = 1000
```

Comment:
to put this number into context, the captains laser (biggest damage dealer that can be acquired through legitimate means) deals about 40 damage per shot.

Didn't realise this was a JRPG

Reply:
Its health at 1000 is equal to Mecha Hitler, double what a spider queen has and four times as much as a cult juggernaut. The aim is to have a big, hulking skeleton, that hits hard but moves slowly. Its also not even functioning yet and no numbers are final. 

We really haven't reached JRPG levels yet.

Reply:
I tend to agree with Martian. Even on station where there are rechargers onhand, spider queens can feel almost comical in how many shots they take. I remember one showing up on the Packed bridge the other night. It couldn't figure out how to get around the smashed monitors and firelocks fortunately so I just sat there for a good two or three minutes repeatedly recharging and shooting it. It wasn't very engaging.

Reply:
We need to fix up the mob AI. If this is a boss mob that can only occur in one place then that's good. Make sure it can't occur through the masks, gold slime cores, antagonist pokeballs, etc.

Reply:
I said more info would come, I only put this up for now so that I could save a bit of progress.

He wont appear anywhere outside of his one little space, that is the intention.